### PR TITLE
Move pypi-helper code to this repository

### PR DESCRIPTION
**Pull request by juanjosevazquezgil**
related to #14 and prefapp/pypi-helper#1

### CODE REVIEW

File: firestarter/common/validations.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    with open(os.path.join(os.path.dirname(__file__), schema_path)) as schema:
+        TASK_MANAGER_SCHEMA = json.load(schema)
+
+    return TASK_MANAGER_SCHEMA
+
+
+def validate_task_manager(task_manager_path: str, schema_path: str) -> dict:
+    with open(task_manager_path, 'r') as task_manager:
+        task_manager_data: dict = yaml.load(
```
Comment (itasahobby) [inline firestarter/common/validations.py:18]: Should not we use `safe_load`?

According to [docs](https://pyyaml.org/wiki/PyYAMLDocumentation)
> Warning: It is not safe to call yaml.load with any data received from an untrusted source! yaml.load is as powerful as pickle.load and so may call any Python function.

File: firestarter/workflows/ci/bash_runner.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    """Builds a bash script from a list of commands.
+
+    Args:
+        commands: A list of strings representing commands.
+        file: A file object where the script will be written.
+    """
+    script: str = "\n".join(["#!/bin/bash"] + commands)
+
+    file.write(script.encode())
+    file.close()
```
Comment (itasahobby) [inline firestarter/workflows/ci/bash_runner.py:15]: I know this was ported from older code but receiving a file descriptor and closing it inside the function call seems a bit off 😅

Comment (juanjosevazquezgil) [reply firestarter/workflows/ci/bash_runner.py]: I agree. This code has been refactored

File: firestarter/common/validations.py (hunk 1 above)
Comment (juanjosevazquezgil) [reply firestarter/common/validations.py]: Updated!

File: firestarter/common/preprocessor.py
[the remark this answers is not in the document]
Comment (juanjosevazquezgil) [reply firestarter/common/preprocessor.py:1]: I don't think adding a dependency is that big of a deal but I don't think it is needed. What we have already fullfils all of our current needs and it's not a big mess of code or anything. Do you think an external preprocessor with more features is needed?

**Review by itasahobby — DISMISSED** [review]
lgtm!

File: firestarter/common/preprocessor.py
[the remark this answers is not in the document]
Comment (itasahobby) [reply firestarter/common/preprocessor.py:1]: Yeah, for just interpolating variables maybe this may get the job done. Let's keep it this way.

**Review by frmadem — APPROVED** [review]
LGTM!

File: firestarter/common/preprocessor.py
[the remark this answers is not in the document]
Comment (alambike) [reply firestarter/common/preprocessor.py:1]: @juanjosevazquezgil do you take a look to https://docs.python.org/3.4/library/string.html#template-strings?
It's in the stdlib

**Pull request merged**